src/voxxml/attributes.ts

```diff
@@ -4,6 +4,31 @@ type Input = 'dtmf' | 'speech' | 'dtmf speech'
 
 type Method = 'GET' | 'POST'
 
+type BankAccountType = 'consumer-checking' | 'consumer-savings' | 'commercial-checking'
+
+type PaymentMethod = 'credit-card' | 'ach-debit'
+
+type TokenType = 'one-time' | 'reusable' | 'payment-method'
+
+type Currency = 'usd'
+
+type PromptFor = 'payment-card-number'
+  | 'expiration-date'
+  | 'security-code'
+  | 'postal-code'
+  | 'bank-account-number'
+  | 'bank-routing-number'
```

Comment:
Should we also add in `payment-processing`?

Reply:
We don't use a prompt for that stage, so I don't think there is a reason for that one to be here.